_rules/audio-is-live-213x3x.md

```diff
@@ -0,0 +1,123 @@
+---
+id: 213x3x
+name: '`audio` element plays live content'
+rule_type: atomic
+description: |
+  This rule checks the content played by an `audio` element is live
+accessibility_requirements:
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Audio output
+acknowledgments:
+  authors:
+    - Carlos Duarte
+---
+
+## Applicability
+
+This rule applies to every `audio` element that is:
+
+- playing; or,
+- has a [play button][] that is [visible][] and [included in the accessibility tree][].
+
+## Expectation 
+
+For each test target, after [seeking][seek] the end of the target element's [media resource][] the [current playback position][] is not the end of the target element's [media resource][].
+
+## Assumptions
+
+_No assumptions._
+
+## Accessibility Support
+
+There are no major accessibility support issues known for this rule.
+
+## Background
+
+- [Understanding SC 1.2.1:Audio-only and Video-only (Prerecorded)](https://www.w3.org/WAI/WCAG21/Understanding/audio-only-and-video-only-prerecorded)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+This `audio` element plays a live media resource by streaming the audio from the device's microphone.
+
+This example requires a browser supporting the experimental MediaStreamTrack API. The browser must be authorized to access a microphone. You can successfully test this example on Google Chrome with the flag --enable-blink-features=WebCodecs,MediaStreamInsertableStreams
```

Comment:
I think we should (also) put that text inside the HTML page of the example itself.
That way, (manual) testers that navigate to the example's page (without necessarily going through the ACT rules page) would be able to see that text and act accordingly.

Reply:
This now seems related to #1679 since "give access to your mic" sounds a bit like a test instruction 🤔

Reply:
* Moved the note to the example so that manual testers are aware of the instructions
* Added a test instructions section after the description of the example